src/components/notification/notification.tsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import classNames from 'classnames'
+import CloseIcon from '../icons/CloseIcon.svg'
+import './notification.less'
+
+type NotificationProps = {
+    id: string
+    icon?: React.ReactNode
+    title?: React.ReactNode
+    subtitle?: React.ReactNode
+    children?: React.ReactNode
+    customCloseButton?: React.ReactNode
+    onClick?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void
+    onClose?: (event: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void
+    closeAltText?: string
+    className?: string
+} & JSX.IntrinsicElements['div']
+
+function Notification({
+    id,
+    icon,
+    title,
+    subtitle,
+    children,
+    customCloseButton,
+    onClick,
+    onClose,
+    closeAltText = 'Close',
+    className,
+    ...rest
+}: NotificationProps) {
+    const titleId = title ? `${id}-title` : null
+    const subtitleId = subtitle ? `${id}-subtitle` : null
+    const contentId = children ? `${id}-content` : null
+    const ariaLabelledBy =
+        children && contentId
+            ? { 'aria-labelledby': contentId }
+            : titleId
+            ? { 'aria-labelledby': titleId }
+            : null
+    const ariaDescribedBy = subtitleId && !children ? { 'aria-describedby': subtitleId } : null
+
+    const notificationContent = (
+        <div id={contentId ?? ''} className="reactist-notification__content">
```

Comment:
Instead of using nullish coalescing here we could just fallback to empty string instead of `null` above? Same for the other ids.

After writing this I asked myself should we add empty ids to the DOM at all?

Reply:
Yeah I guess we should spread it like the aria attributes